tests/integration/find-references-identity-redeclared-local.test.ts

```
@@ -0,0 +1,192 @@
+        scope_resolver = new ScopeResolver();
+        forward_scope_resolver = new ForwardScopeResolver(scope_resolver, {
+            max_forward_depth: 10,
+        });
+        scope_resolver.set_forward_scope_resolver(forward_scope_resolver);
+        scope_resolver.set_dependency_graph(dep_graph);
+
+        provider = new ReferencesProvider(scope_resolver);
+    });
+
+    afterEach(() => {
+        try { scope_resolver?.dispose(); } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)